api/pkg/cli/cmd/install/install_test.go

```diff
@@ -59,6 +59,36 @@ var resVersion = &res.ResourceVersionData{
 	},
 }
 
+var deprecated = true
+var resDeprecatedVersion = &res.ResourceVersionData{
+	ID:                  11,
+	Version:             "0.2",
+	DisplayName:         "foo-bar",
+	Description:         "v0.2 Task to run foo",
+	Deprecated:          &deprecated,
```

Comment:
Can't we directly use  `true` here ??

Reply:
No, we can't use directly

Reply:
Why ??

Reply:
it's an optional field so it takes an address

Reply:
Ahh right I got it, it's the optional value and hence needs to be a pointer 👍🏻